### ISSUE TITLE
Transfer to another domain or server       

### ISSUE THREAD

**Issue by fbojan** [YetiForceCompany/YetiForceCRM#10007]
EXAMPLE: I install one cloud for my company YETIFORCE. http://cloud.si/yetiforce. After a certain time, I would like to download the program to a local server or other domain. I changed ***** config / Db.php, but somehow it does not work properly or did I forget something else in the sql base. ??? 

THANK YOU

**Comment by vovpff** [comment]
You need also upload dump of database to new server.

**Comment by rskrzypczak** [comment]
If you change the domain you also have to modify the $site_URL variable in the config/Main.php file.

**Comment by alagmouch** [comment]
I had to also adjust the **root_directory** variable in **config/Modules/OSSMail.php** for RoundCube to work.

**Comment by fbojan** [comment]
I've followed your instructions and it works. Thank you all.

**Comment by Amypate85** [comment]
Can you please explain in layman's terms? lol I want my email to end in @comanyName.com

**Comment by mariuszkrzaczkowski** [comment]
https://yetiforce.com/en/knowledge-base/documentation/implementer-documentation/item/migration-to-a-new-server

**Comment by Amypate85** [comment]
Thank you so much!!

**Comment by naturogem** [comment]
Hello, since the domain change (no server change, just the domain that is changed), RoundCube no longer works. I read above that the root_directory should be adjusted in config/Modules/OSSMail.php but I don't understand what it is.

Before domain change in $site_URL of config/Main.php file everything was working fine. When I put the old domain back, RoundCube is still knocked out. I do not know what to do ...

I am under version 6.4 of YetiForce

Thanks for your help.
<img width="1266" alt="Capture d’écran 2023-01-25 à 10 59 26" src="https://user-images.githubusercontent.com/123548364/214534767-833a88df-3545-419c-b7af-4e9a4ec19688.png">

**Comment by mariuszkrzaczkowski** [comment]
logs.. ?

**Comment by naturogem** [comment]
I just fixed the problem just now. I had also changed the name of the folder to root, which made RoundCube trouble getting the files it needed. By putting the original name of the folder back, everything works again.
